static/src/js/components/PermissionForm/PermissionForm.jsx

```diff
@@ -546,6 +560,16 @@ const PermissionForm = ({ selectedCollectionsPageSize }) => {
       )
     }
 
+    if (providerIdCheckFails(providerId, conceptIds)) {
+      errorLogger('Error multiple providers in collections', 'PermissionForm: providerIdCheck')
+      addNotification({
+        message: 'Error multiple providers in collections',
```

Comment:
This error message isn't actionable, if we're going to display a message to the user, they need to be able to take an action to make the error go away. This message doesn't tell them what they need to do.

Reply:
Suggestions?  This is the use case for creating a new ACL, where we don't have the provider id, until submission.   The old code was should just fail silently and showing a passing message even thought there were collections not in the same provider as the ACL.   We can make the message longer, if you have a better suggestion.

Reply:
I missed your comment, I'll add the provider id dropdown directly into the form.

Reply:
Actually, adding the provider id combo box doesn't really prevent this, as they can still select multiple collections with multiple provider ids.   So I'm going to keep the check, but if you have some suggestions as to how to word the error, that would be great.

Reply:
If CMR doesn't prevent this, we shouldn't be either. If there is a situation that we are allowing the user to do something that produces an error, then we need to _prevent it_ from happening. If CMR doesn't care about this, then we shouldn't be adding validation on top of CMR. 

IF CMR doesn't want this to happen, then CMR needs to make it return an error, and we can then file a ticket to prevent the user from creating data that will produce an error.

Reply:
Ok removed.